src/navigation/breadcrumbs/breadcrumbsUtils.js

```diff
@@ -0,0 +1,202 @@
+import {
+    get,
+    uniqBy,
+    find,
+} from 'lodash';
+
+const extractRoute = (route, prefix, shouldAddRoute = true) => {
+    const path = route.props && route.props.path ? route.props.path : route.path;
+    let paths = [];
+
+    if (!path) {
+        if (Array.isArray(route)) {
+            route.forEach((r) => {
+                paths = paths.concat(extractRoute(r, prefix, shouldAddRoute));
+            });
+
+            return paths;
+        }
+        // eslint-disable-next-line no-use-before-define
+        return extractChildRoutes(route, prefix);
+    }
+
+    const currentPath = (`${prefix || ''}${path}`.replace(/\/+/g, '/'));
+    const pathString = `${currentPath.startsWith('/') ? '' : '/'}${currentPath}`;
+    let title = get(route, 'title');
+    const couldItBeHome = path === '/' && !title;
+
+    if (couldItBeHome) {
+        title = 'Home';
+    }
```

Comment:
Seems more like we consider this `itDefinitivelyIsHomeAbsolutelyForSureNoDoubtsAboutIt`! :trollface: 
No action ... just commenting! =)